api/server/sdk/rest_gateway.go

```diff
@@ -90,7 +91,7 @@ func (s *sdkRestGateway) Stop() {
 
 // restServerSetupHandlers sets up the handlers to the swagger ui and
 // to the gRPC REST Gateway.
-func (s *sdkRestGateway) restServerSetupHandlers() (*http.ServeMux, error) {
+func (s *sdkRestGateway) restServerSetupHandlers() (http.Handler, error) {
```

Comment:
We don't lose any functionality by changing the '*http.ServeMux' to 'http.Handler'?

Reply:
ServeMux is an implementation of http.Handler, so it should be okay.